ods_ci/tests/Tests/400__ods_dashboard/416__ods_dashboard_settings_accelerator_profiles.robot

```diff
@@ -0,0 +1,154 @@
+*** Settings ***
+Resource        ../../Resources/Common.robot
+Resource        ../../Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource
+Resource        ../../Resources/ODS.robot
+Suite Setup     Setup Settings Accelerator Profiles
+Suite Teardown  Teardown Settings Accelerator Profiles
+
+
+*** Variables ***
+${ACC_DISPLAY_NAME}=    qe_create_ap_
+${ACC_NAME}=    qecreateap
+${ACC_IDENTIFIER}=    nvidia.com/gpu
+${ACC_DESCRIPTION}=    description example
+${ACC_ENABLED}=    True
+${ACC_TOLERATION_OPERATOR}=    Exists
+${ACC_TOLERATION_EFFECT}=    PreferNoSchedule
+${ACC_TOLERATION_KEY}=    my_key
+${ACC_TOLERATION_VALUE}=    my_value
+${ACC_TOLERATION_SECONDS}=    15
+
+
+*** Test Cases ***
+Create An Accelerator Profile From "Accelerator Profiles" Administration UI
+    [Documentation]    Create an Accelerator Profile instance from the Administration UI and verify it's content
+    [Tags]  ODS-XXX
+    ...     Smoke
+    Open ODS Dashboard With Admin User
+    Navigate To Page    Settings    Accelerator profiles
+    Click On Create Accelerator Profile Button
+    Create An Accelerator Profile Via UI   ${ACC_DISPLAY_NAME}01   ${ACC_IDENTIFIER}
```

Comment:
From the list of the `qe_create_ap_*` files attached to this PR, it looks like the one for this case is missing?

Reply:
This one is going to create the instance using the UI, and those files are used for creating instances using OC  (in order to provision some specific tests). So no need of file for this test.